src/js/modules/sections/components/SectionPage.js

```diff
@@ -40,31 +33,28 @@ const styles = {
     width: '100%',
     margin: '0px',
   },
-  Horizontal: {
+  listOfSubsections: {
     display: 'inline',
```

Comment:
What you're referencing is not a list; it is a list item.

Change the class name.